Extensions/BitmapText/bitmaptextruntimeobject-pixi-renderer.js

```diff
@@ -55,20 +51,17 @@ gdjs.BitmapTextRuntimeObjectPixiRenderer = function (
       .getPIXITexture(bitmapTextureResourceName);
 
     // Get the bitmap font file and use the texture for generate the bitmapFont (PIXI.BitmapFont) and return the fontName ready to use.
-    runtimeScene
+    const bitmapFont = runtimeScene
       .getGame()
       .getBitmapFontManager()
-      .loadBitmapFont(bitmapFontResourceName, texture)
-      .then((fontName) => {
-        this._pixiObject.fontName = fontName;
-        this.updatePosition();
-      })
-      .catch((error) => {
-        console.error('Error while loading a bitmapFont resource:', error);
-      });
+      .getBitmapFontFromData(bitmapFontResourceName, texture);
+    this._pixiObject.fontName = bitmapFont.font;
+    this._pixiObject.fontSize = bitmapFont.size;
+
+    this.updatePosition();
   } else {
```

Comment:
Note that this if/else is now actually useless because we don't hack anymore the constructor to run them on already constructed object. So you can keep only the code inside the if branch :)